app/src/main/java/com/codesoom/http/HttpRequest.java

```diff
@@ -0,0 +1,63 @@
+package com.codesoom.http;
+
+import com.codesoom.exception.MethodNotExistException;
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+public class HttpRequest {
+    private final HttpMethod method;
+    private final String path;
+    private final String body;
+
+    public HttpRequest(HttpMethod method, String path, String body) {
+        this.method = method;
+        this.path = path;
+        this.body = body;
+    }
+
+    public HttpRequest(HttpExchange exchange) {
+        method = HttpMethod.of(exchange.getRequestMethod());
+        path = exchange.getRequestURI().getPath();
+        InputStream in = exchange.getRequestBody();
+        body = new BufferedReader(new InputStreamReader(in))
+                .lines()
+                .collect(Collectors.joining("\n"));
+    }
+
+    public static boolean isValidRequest(HttpExchange exchange) {
```

Comment:
이름이 `isValid`여도 충분할 것 같아요.

```suggestion
    public static boolean isValid(HttpExchange exchange) {
```

Reply:
b90c751694972615b28d8147e540658bdedc78f9 수정했습니다!